src/main/java/spireMapOverhaul/zones/thefog/events/StandingStones.java

```diff
@@ -0,0 +1,88 @@
+package spireMapOverhaul.zones.thefog.events;
+
+import com.badlogic.gdx.math.MathUtils;
+import com.megacrit.cardcrawl.cards.DamageInfo;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.events.AbstractImageEvent;
+import com.megacrit.cardcrawl.localization.EventStrings;
+import com.megacrit.cardcrawl.rooms.AbstractRoom;
+import spireMapOverhaul.rewards.AnyColorCardReward;
+import spireMapOverhaul.zones.thefog.relics.StoneFragment;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+import static spireMapOverhaul.SpireAnniversary6Mod.makeImagePath;
+import static spireMapOverhaul.util.Wiz.adp;
+
+public class StandingStones extends AbstractImageEvent {
```

Comment:
Am I right that you haven't done anything to make this event show up more often? In other words, it will just be in the pool of events while you're in the zone, no more or less likely to show up than any of the other events for the act the player is in.

Since the whole zone will be ? nodes and the player is likely to encounter some events in the zone, that's fine if you want it to work like that. On the other hand, if you want this to show up a bit more often, you can add something like this to your zone class:
```

    @Override
    public float zoneSpecificEventRate() {
        return 0.3f;
    }
```
That would give a 30% chance per event that you get a zone-specific event; I picked 30% as something that seemed like a reasonable bump to the normal probability without guaranteeing you see this event every time you're in the zone.

Up to you though, I'm making this comment just to call out the option to do this.